src/Traits/TButtonIcon.php

```diff
@@ -20,10 +20,14 @@ trait TButtonIcon
 	/**
 	 * Set icon
 	 * @param string $icon
+	 * @param string $content
 	 */
-	public function setIcon($icon)
+	public function setIcon($icon, $content = '')
 	{
-		$this->icon = $icon;
+		$this->icon = (object)[
```

Comment:
Can you please new some new object? For example `IcoData` that will have two properties icon and content? it will be better typed that a `stdClass` instance. Thank you. :)